snapcraft.yaml

```diff
@@ -0,0 +1,53 @@
+name: check-sieve
+license: MIT
+version: "0.6"
+summary: Syntax checker for mail sieves
+description: |
+  Makes checking mail sieve syntax easy and painless.
+  Because breaking your sieve in production sucks.
+
+type: app
+base: core22
+grade: stable
+compression: xz
+
+confinement: strict
+adopt-info: check-sieve
+
+parts:
+  check-sieve:
+    plugin: make
+    source: https://github.com/dburkart/check-sieve.git
+    build-packages:
+      - g++
+      - gcc
+      - make
+      - bison
+      - flex
+      - python3-dev
```

Comment:
Bison, flex, and python should only be required for dev, not to simply build from source. Not sure if that matters at all.

Reply:
True. The build task is not running Bison, Flex, or the tests in this context since they're cached in the repository.

They can be safely dropped for a bit of build performance. And that performance does count when it occurs in a GitHub action (e.g., runner seconds).

Reply:
Removed `bison`, `flex`, and `python3-dev`.